### PR TITLE
[JENKINS-33306] Stack trace printed when repo:status privilege missing

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-33306](https://issues.jenkins-ci.org/browse/JENKINS-33306)

### Example

```
Started by user anonymous
Connecting to GitHub using recena/****** (MyPAT)
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from 2 remote Git repositories
 > git config remote.origin.url https://github.com/seville-jam/workflow-helloworld.git # timeout=10
Fetching upstream changes from https://github.com/seville-jam/workflow-helloworld.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/var/folders/y_/9z0ktk9d2lxcs18kw7l10h9w0000gn/T/git4664648971215722216.credentials # timeout=10
 > git fetch --tags --progress https://github.com/seville-jam/workflow-helloworld.git +refs/heads/*:refs/remotes/origin/*
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin1.url https://github.com/seville-jam/workflow-helloworld.git # timeout=10
Fetching upstream changes from https://github.com/seville-jam/workflow-helloworld.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/var/folders/y_/9z0ktk9d2lxcs18kw7l10h9w0000gn/T/git7045466587526744178.credentials # timeout=10
 > git fetch --tags --progress https://github.com/seville-jam/workflow-helloworld.git +refs/pull/*/merge:refs/remotes/origin/pr/*
 > git config --local --remove-section credential # timeout=10
Checking out Revision a8e55ab71a968d6dcf945172f0e3a5c64190cdb5 (PR-4)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a8e55ab71a968d6dcf945172f0e3a5c64190cdb5
 > git rev-list a8e55ab71a968d6dcf945172f0e3a5c64190cdb5 # timeout=10
[Pipeline] Allocate node : Start
Running on master in /Users/recena/Development/projects/github-branch-source-plugin/work/jobs/JENKINS-33306-1/jobs/workflow-helloworld/branches/PR-4/workspace
[Pipeline] node {
[Pipeline] echo
Hello World
[Pipeline] } //node
[Pipeline] Allocate node : End
[Pipeline] End of Pipeline

Could not update commit status, please check if your scan credentials belong to a member of the organization or a collaborator of the repository and repo:status scope is selected

Finished: SUCCESS
```

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java [hunk 1]
```diff
@@ -275,46 +275,46 @@ private void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepos
         }
         listener.getLogger().format("%n  %d branches were processed%n", branches);
 
-            listener.getLogger().format("%n  Getting remote pull requests...%n");
```
Comment (recena) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: I've tried to remove a wrong indentation....

**Comment by jglick** [comment]
Hmm, [whitespace-ignoring diff](https://github.com/jenkinsci/github-branch-source-plugin/pull/40/files?w=1) not working any more, perhaps after the big PR UI overhaul. Hard to follow this patch.

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java (hunk 1 above)
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: If that is the _only_ think you are changing, better to do a separate PR (or simply direct commit) that fixes the indentation, so as not to confuse PR reviews.

**Comment by recena** [comment]
@jglick I've reverted the wrong indentation. I'll send a separated PR for that.

**Comment by recena** [comment]
@jglick https://github.com/jenkinsci/github-branch-source-plugin/pull/41

**Comment by jglick** [comment]
Clearer, thanks, :bee:

**Comment by andresrc** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done

**Comment by recena** [comment]
Thank @jglick and @andresrc
